packages/mui-material/src/Select/Select.test.js

```diff
@@ -938,6 +939,62 @@ describe('<Select />', () => {
         expect(onChange.secondCall.returnValue).to.deep.equal({ name: 'age', value: [30, 10] });
       });
     });
+
+    it('should apply multiple class', () => {
+      const { container } = render(
+        <Select multiple open value={[10, 30]}>
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>,
+      );
+
+      expect(container.querySelector(`.${classes.select}`)).to.have.class(classes.multiple);
+    });
+
+    it('slots overrides should work', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+
+      const selectStyle = {
+        marginLeft: '10px',
+        marginTop: '10px',
+      };
+
+      const multipleStyle = {
+        marginTop: '14px',
+      };
+
+      const theme = createTheme({
+        components: {
+          MuiSelect: {
+            styleOverrides: {
+              select: selectStyle,
+              multiple: multipleStyle,
+            },
+          },
+        },
+      });
+
+      const { container } = render(
+        <ThemeProvider theme={theme}>
+          <Select open value={['first']} multiple>
+            <MenuItem value="first" />
+            <MenuItem value="second" />
+          </Select>
+        </ThemeProvider>,
+      );
+
+      const combinedStyle = merge(selectStyle, multipleStyle);
```

Comment:
```suggestion
      const combinedStyle = { ...selectStyle, ...multipleStyle };
```
I think this should be enough.

Reply:
Same as `NativeSelect`.

Reply:
Nice. Good call out. Updated 😄